GhostProjectBackend/settings.py

```diff
@@ -74,11 +88,36 @@
 # Database
 # https://docs.djangoproject.com/en/4.2/ref/settings/#databases
 
+# DATABASES = {
```

Comment:
We should delete unused code, there is no need for this do be committed. Apply this everywhere.